{{cookiecutter.__src_folder_name}}/tests/conftest.py

```diff
@@ -85,16 +90,21 @@ def mock_functions_env():
     pass
     {% endif %}
 
+{% if cookiecutter.project_backend == "django" %}
 @pytest.fixture(scope="session")
 def live_server_url(live_server):
     """Returns the url of the live server"""
-    {% if cookiecutter.project_backend == "django" %}
     return live_server.url
-    {% endif %}
-    {% if cookiecutter.project_backend == "fastapi" %}
+{% endif %}
+{% if cookiecutter.project_backend == "fastapi" %}
+@pytest.fixture(scope="session")
+def live_server_url(live_server):
+    """Returns the url of the live server"""
     return 'http://localhost:8000'
-    {% endif %}
-    {% if cookiecutter.project_backend == "flask" %}
+{% endif %}
+{% if cookiecutter.project_backend == "flask" %}
+@pytest.fixture(scope="function")
```

Comment:
There were enough differences for this one that I broke it out- the scope has to be function, and it has a dependency on app - that ensure it's working inside an app context.